src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/OrderSummary/OrderSummaryController.ts

```diff
@@ -89,6 +89,14 @@ module Orckestra.Composer {
                             console.error('Error while proceeding to Checkout', reason);
                             ErrorHandler.instance().outputErrorFromCode('ProceedToCheckoutFailed');
                         });
+                    },
+                    removeInvalidLineItems() {
+                        this.Mode.Loading = true;
+                        self.orderSummaryService.cleanCart()                
+                            .then(this.cartService.invalidateCache())
```

Comment:
should be **self**.cartService.invalidateCache(); and move before the cleanCart()